packages/plugins/src/DocumentAssetsPlugin.ts

```diff
@@ -97,22 +97,17 @@ const DocumentAssetsPlugin: PluginType<Page, DocumentAssetsPluginOptions> = {
 
       let globbedImageDirs;
       try {
-        await fsExtra.access(resolvedSrcDir);
         globbedImageDirs = await glob(assetSubDir, {
           cwd: resolvedSrcDir,
           onlyDirectories: true
         });
-      } catch (err) {
-        if (err.code === 'ENOENT') {
-          console.warn(`Warning: Directory ${resolvedSrcDir} does not exist.`);
-        } else {
-          console.error(`Error globbing ${assetSubDir} in ${srcDir}:`, err);
-          throw err;
+        if (globbedImageDirs.length === 0) {
+          console.warn(`Warning: No entries found for ${assetSubDir}. It may not exist.`);
+          continue;
         }
-      }
-
-      if (!globbedImageDirs || globbedImageDirs?.length === 0) {
-        continue;
+      } catch (err) {
+        console.error(`Error globbing ${assetSubDir} in ${srcDir}:`, err);
+        throw err;
```

Comment:
Throw a plugin error and it can be tracked and displayed in the admin APIs 

```throw new PluginError(e.message, page.fullPath);```

Adding fullPath of the page makes it easy to see what page has the issue but its optional and might not apply here